perfkitbenchmarker/providers/openstack/os_disk.py

```diff
@@ -12,107 +12,155 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import json
 import logging
-import time
-
 
+from perfkitbenchmarker import disk
 from perfkitbenchmarker import errors
 from perfkitbenchmarker import flags
-from perfkitbenchmarker import disk
+from perfkitbenchmarker import vm_util
 from perfkitbenchmarker.providers.openstack import utils as os_utils
 
+REMOTE_VOLUME_DEFAULT_SIZE_GB = 50
+
 FLAGS = flags.FLAGS
 
 
+def CreateVolume(resource, name, image=None):
+  """Creates a remote (Cinder) block volume."""
+  vol_cmd = os_utils.OpenStackCLICommand(resource, 'volume', 'create', name)
+  vol_cmd.flags['availability-zone'] = resource.zone
+  if image:
+    vol_cmd.flags['image'] = image
+    vol_cmd.flags['size'] = GetImageMinDiskSize(resource, image)
+  if FLAGS.openstack_volume_size:
```

Comment:
Good catch. Actually, so in this case, we want to set it to the image min_size required if the volume is being used as a bootable volume. Otherwise, it should just default to either the flag or the default volume size.